.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags: 
+      - '[0-9]+.[0-9]+.[0-9]+'
+
```

Comment:
Something like below ?

```
on:
  push:
    tags:
      - '*'
```

Reply:
No,the idea is to trigger only onsemantic version tag.

Reply:
The releases seem to be prefixed with a `v`. Will this trigger in that case ?

Reply:
The description of the PR says that prefix `v` was removed from the release version.